src/tools/cephfs/setup.py

```diff
@@ -14,6 +14,7 @@
         'cephfs',
         'cmd2',
         'colorama',
+        'tox',
```

Comment:
this is not right, cephfs-shell doesn't require tox to work, it shouldn't be required

Reply:
I will remove it.